subsys/logging/log_output_syst.c

```diff
@@ -308,7 +308,7 @@ static void write_d32(struct mipi_syst_handle *systh, uint32_t v)
 }
 
 #if defined(MIPI_SYST_PCFG_ENABLE_64BIT_IO)
-static void write_d64(struct mipi_syst_handle *systh, uint64_t v)
```

Comment:
this should be in its own commit explaining why you are changing this.

Reply:
Not that there is a filter in place for 64 bit architectures, this is not needed. Hence changed it back to uint64_t.